core/src/main/java/org/jdbi/v3/core/statement/Binding.java

```diff
@@ -151,19 +151,75 @@ public Optional<Argument> findForPosition(int position) {
 
     @Override
     public String toString() {
-        String positionalsDescription = positionals.entrySet().stream()
-            .map(x -> x.getKey().toString() + ':' + unwrap(x.getValue()))
-            .collect(Collectors.joining(","));
-
-        String namedDescription = named.entrySet().stream()
-            .map(x -> x.getKey() + ':' + unwrap(x.getValue()))
-            .collect(Collectors.joining(","));
-
-        String found = namedArgumentFinder.stream()
-            .map(Object::toString)
-            .collect(Collectors.joining(","));
+        return toString(Integer.MAX_VALUE);
+    }
 
-        return "{positional:{" + positionalsDescription + "}, named:{" + namedDescription + "}, finder:[" + found + "]}";
```

Comment:
can you comment a bit what that change is? The original binding tostring is already convoluted, what does this do?

Reply:
Just adding an "early out" maximum so we can render a bindings list with e.g. 1000 parameters without blowing up our heap